### PR TITLE
Consolidate and update licensed list to specify resources users can manage in broswer

### LINKED ISSUE #1925: License activated features are missing some web ui functionality
## This is an issue with:

- [ ] Bug (site functionality or styling)
- [ ] Errata (fix needed for doc content)
- [ ] New content (guide wanted)
- [x] Update (Add missing or refresh existing content)
- [ ] Enhancement (add new site functionality)

## Expected Behavior
[License activated features](https://docs.sensu.io/sensu-go/5.14/getting-started/enterprise/#license-activated-features-in-sensu-go) should contain all license activated features.

## Current Behavior
[License activated features](https://docs.sensu.io/sensu-go/5.14/getting-started/enterprise/#license-activated-features-in-sensu-go) is missing web ui resource management for silences, mutators and filters.
```
-> Manage your monitoring checks from your browser: Create, edit, and delete checks using the Sensu web UI.
-> Manage event handlers from your browser: Create, edit, and delete handlers using the Sensu web UI.
```

## Possible Solution
Add bullet points for those resources, or consolidate all web ui resource management features into one point.

## Context
Although the idea of license activated features is changing, we should maintain an accurate list of them, especially for users compiling from source.

### PR DESCRIPTION
**Pull request by hillaryfraley**
## Description
- Consolidates list of resources users can manage in web UI w/license
- Adds silences, mutators, and filters to list of resources users can manage in web UI w/license

## Motivation and Context
Closes https://github.com/sensu/sensu-docs/issues/1925

### CODE REVIEW

**Review by nikkictl — COMMENTED** [review]
Not sure if this needs a rebase or not, but I'm seeing some changes unrelated to issue.

**Comment by hillaryfraley** [comment]
@nikkixdev How far back should this change go in the docs? Looks like:

- Added the license-activated features page in 5.2, but only checks are listed for web UI management
- Handlers are added to the list in 5.9

Do you know when (i.e. at which version) the silences, mutators, and filters were added?

**Comment by nikkictl** [comment]
Filters and mutators were added in [5.11.0](https://docs.sensu.io/sensu-go/5.14/release-notes/#5-11-0-release-notes), and it looks like silences were added in [5.2.0](https://docs.sensu.io/sensu-go/5.14/release-notes/#5-2-0-release-notes) (that's unclear though).

File: content/sensu-go/5.15/sensuctl/reference.md [hunk 1]
```diff
@@ -729,7 +729,7 @@ SET SENSU_REFRESH_TOKEN=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.x.x
 SET SENSU_TRUSTED_CA_FILE=
 SET SENSU_INSECURE_SKIP_TLS_VERIFY=true
 REM Run this command to configure your shell:
-REM 	@FOR /f "tokens=*" %i IN ('sensuctl env --shell cmd') DO @%i
+REM   @FOR /f "tokens=*" %i IN ('sensuctl env --shell cmd') DO @%i
```
Comment (hillaryfraley) [inline content/sensu-go/5.15/sensuctl/reference.md]:
```suggestion
REM    @FOR /f "tokens=*" %i IN ('sensuctl env --shell cmd') DO @%i
```

Comment (hillaryfraley) [inline content/sensu-go/5.15/sensuctl/reference.md]:
```suggestion
REM 	@FOR /f "tokens=*" %i IN ('sensuctl env --shell cmd') DO @%i
```

Comment (nikkictl) [inline content/sensu-go/5.15/sensuctl/reference.md]: What editor are you using?

**Review by nikkictl — COMMENTED** [review]
Does it make sense to add the other resources to the docs retroactively for past versions?

**Comment by hillaryfraley** [comment]
Yeah, I thought the 5.2 release notes might be saying you could silence events as something separate from the create/update/delete control over silences.

I see this at [5.5](https://docs.sensu.io/sensu-go/5.14/release-notes/#5-5-0-release-notes): "The Sensu web UI now displays silences created by LDAP users."

**Comment by hillaryfraley** [comment]
Yes, I'll fix the previous versions...I'm just trying to figure out when to add the silences.

**Comment by hillaryfraley** [comment]
Cameron clarified that 5.2.0 sounds right for silences. Updating the PR now.

**Review by nikkictl — COMMENTED** [review]
@jamesdphillips says silence management predates GA, which makes sense why an entry is not in the changelog. We can add it to the license activated features through 5.0.0.

**Comment by hillaryfraley** [comment]
Clarification from Cameron:
cwj: 12:55 PM
I don't think that managing silencing entries via the web UI is a commercial feature
meaning, its part of the open source core afaik
silencing entries are more ad-hoc and ephemeral than other resources, so I think we've treated them differently

**Review by nikkictl — APPROVED** [review]
:shipit:

**Pull request merged**